lib/mimdb/actors.ex

```diff
@@ -0,0 +1,104 @@
+defmodule Mimdb.Actors do
```

Comment:
Your probably considered this, but should always think about the name of the Context. Should it be a higher level concept than Actor? There is no 'right' answer hear, just a feeling of cohesion that you will develop to understand what entities naturally belong with each other and what you should call that Context as a result.

Reply:
Thinking again on this, what do you think about name as context `Movies`? Because it's an Movie's actor

Reply:
Yea, I have a feeling that much of what we create (maybe with the exception of User) could logically be in Movies Context